lib/rodauth/features/login.rb

```diff
@@ -22,7 +22,10 @@ module Rodauth
     auth_cached_method :multi_phase_login_forms
     auth_cached_method :login_form_footer
 
-    auth_value_methods :login_return_to_requested_location_path
+    auth_value_methods(
+      :login_response,
```

Comment:
`login_response` makes sense as an auth method, but not as an auth value method.

Reply:
can you clarify what's the distinction? FTR I've been defining only `auth_value_method`s in `rodauth-oauth`, and perhaps i shouldn't, but looking at the source code it seems that they're almost the same, except value methods allow for values instead of routines?

I'll do the change meanwhile.

Reply:
`auth_value_method` is for methods where it may make sense to assign a static value, that does not depend on runtime state.  That's not likely for `login_response`, since by default, it redirects (throws), it does not return a value.  It's also inconsistent with the `response` feature method, since that adds an `auth_method` and not an `auth_value_method`.